index.html

```diff
@@ -0,0 +1,981 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    <link rel="icon" href="/images/logo.png" />
```

Comment:
## Images not loading
- [ ] The pages images are not loading correctly. This could be a case of how we are referencing our images. There is a universal way of ensuring images and files getting referenced gets to be accurately referenced by prepending their paths with a period sign. Have a look at one of my suggestion Below 👇🏾 
```suggestion
    <link rel="icon" href="./images/logo.png" />
```
|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/AlexJustAlex2020/newyorktimes/nytimes-layout/index.html)|
|:--:|:--:|
|![source](https://user-images.githubusercontent.com/22345961/112547201-7ffb8680-8dcb-11eb-83c8-5a03a04209d4.png)|![clone](https://user-images.githubusercontent.com/22345961/112547260-96094700-8dcb-11eb-97ef-532a0dbc9833.png)|